eq-author/src/App/Submission/Design/SubmissionEditor/index.js

```diff
@@ -29,14 +30,13 @@ const SectionTitle = styled.h2`
 `;
 
 const SectionDescription = styled.p`
-  margin: 0.1em 0 1em;
-  color: ${({ theme }) => theme.colors.textLight};
+  margin: 0.2em 0 2em;
```

Comment:
```suggestion
  margin: 0.1em 0 1em;
```
Reduces the size of the margin